README.md

```diff
@@ -46,8 +50,12 @@ parameters:
   wisdom: !request 'https://ron-swanson-quotes.herokuapp.com/v2/quotes'
 ```
 
+Request with basic authentication:
 ```yaml
 parameters:
   wisdom: !request
-    url: https://ron-swanson-quotes.herokuapp.com/v2/quotes
+    url: https://private.endpoint.com/
+    auth: basic
+    user: MyUsername
+    password: !ssm /app/prod/MyUserPwd
```

Comment:
This will only work after we release our latest version.

Reply:
oh yeah, changed it back to a ho-hum string.l